### PR TITLE
do not recommend Implicit for testing

### PR DESCRIPTION
**Pull request by aaronpk**
Postman handles the authorization code exchange as well as PKCE so it's just as easy to use for this guide.

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md [hunk 1]
```diff
@@ -10,24 +10,26 @@ After you have the following, you can get an access token and make a request to
...
 
     * **Callback URL** &mdash; Define the callback location where Okta returns the token after the user finishes authenticating. This URL must match one of the redirect URIs that you configured in the <GuideLink link="../create-oauth-app">Create an OAuth 2.0 app in Okta</GuideLink> section.
-    * **Auth URL** &mdash; Enter the authorization endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/authorize?nonce=WBYA`
-        > **Note:** There isn't an entry in the dialog box for a `nonce` parameter. But, you can append it to the Auth URL here. Use any value for `nonce`. This is a string that is included in the returned ID token. It associates a client session with an ID token and mitigates replay attacks. In this example, we aren't requesting the `id_token` response type, so an ID token isn't returned.
+    * **Auth URL** &mdash; Enter the authorization endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/authorize`
+    * **Access Token URRL** &mdash; Enter the token endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/token`
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md:26]:
```suggestion
    * **Access Token URL** &mdash; Enter the token endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/token`
```

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md (hunk 1 above)
Comment (aaronpk) [reply packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md]: my damn keyboard keeps double-pressing keys!

**Comment by jakubvul-okta** [comment]
I'd like to get @venkatviswanathan-okta 's approval on this before it goes out.

**Review by venkatviswanathan-okta — APPROVED** [review]
Tested with the updated steps and it works. 🚢

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**